test/api/v3/integration/user/POST-user_read_card.test.js

```diff
@@ -0,0 +1,38 @@
+import {
+  generateUser,
+  translate as t,
+} from '../../../../helpers/api-integration/v3';
+
+describe('POST /user/read-card/:cardType', () => {
+  let user;
+  let cardType = 'greeting';
+
+  beforeEach(async () => {
+    user = await generateUser();
+  });
+
+  it('returns an error when cardType not provided', async () => {
+    await expect(user.post('/user/read-card'))
+      .to.eventually.be.rejected.and.to.eql({
+        code: 404,
+        error: 'NotFound',
```

Comment:
I think it returns a 404 because the user/read-card route without a parameter doesn't exist